src/lein_monolith/task/each.clj

```diff
@@ -236,14 +237,26 @@
         opts (:opts ctx)
         subproject (get-in ctx [:subprojects target])
         results (delay {:name target
-                        :elapsed (/ (- (System/nanoTime) start) 1000000.0)})]
+                        :elapsed (/ (- (System/nanoTime) start) 1000000.0)})
+        marker (:changed opts)
+        save? (:save-fingerprints opts)
```

Comment:
Naming: this could be more specific, like `save-print?` However, you only use it once so I'm not sure you even need to pull it out here.